File: java/1005.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        double a, b, media;
        
        a = in.nextDouble();
        b = in.nextDouble();
        media = ((a * 3.5) + (b * 7.5)) / 11;
        System.out.printf("MEDIA = %.5f\n", media);
    }
}

File: C++/1005.cpp
#include <iostream>
 
using namespace std;
 
int main() {
    double a = 0, b = 0, media = 0;
    
    cin >> a;
    cin >> b;
    media = ((a * 3.5) + (b * 7.5)) / 11;
    printf("MEDIA = %.5f\n", media);
    return 0;
}